### ISSUE TITLE
figure sizes need to be reduced

### ISSUE THREAD

**Issue by jstac** [QuantEcon/lecture-python-programming.rst#34]
Apart from the first one, all the figures on this page are too big: https://python-programming.quantecon.org/about_py.html.

@shlff @Harveyt47 @najuzilu we talked about how to fix this for a long time -- maybe too long, so nothing got done :-)

I suggest we just reduce the size of them using the `:scale:` argument to the `figure` directive.  Would someone be able to help with this?

**Comment by Harveyt47** [comment]
Hey @jstac is this also to do with the figure size of figures generated by code across the lectures. Or just PNGs, because I remember something similar in [#767](https://github.com/QuantEcon/lecture-source-py/issues/767). I'm uncertain if this overlaps.

**Comment by jstac** [comment]
This is separate because it just pertains to PNG files, or whatever kinds of files we're loading into the page via `.. figure::`

@DrDrij at one point said something about how we should use an image manipulation program to change the size of the figures but I think this is unworkable for us.  I suggest we just alter the size via `:scale:`.

**Comment by shlff** [comment]
Thanks, @jstac . Hi mates, we also need to update the second figure ([pandas vs matlab](https://github.com/QuantEcon/lecture-python-programming/blob/master/source/_static/lecture_specific/about_py/pandas_vs_matlab.png)) of that lecture (https://python-programming.quantecon.org/about_py.html) to 2020 version.

**Comment by najuzilu** [comment]
I have utilized `:scale:` in [quantecon-example](https://github.com/ExecutableBookProject/quantecon-example) (see `book/docs/`). I think it would be easy to copy-paste + minor edits into the actual `rST` files if @jstac  thinks the size dimensions in the book example are what we're aiming.

**Comment by jstac** [comment]
Yes, that sounds great to me @najuzilu, thanks.  We just need to use :scale: and that version of the same material has the right scale numbers.

@shlff yes please!

**Comment by najuzilu** [comment]
It looks like copy-pasting the scale parameter doesn't necessarily scale them to the same size as in the [quantecon-example](https://github.com/ExecutableBookProject/quantecon-example). I just rendered them locally and played around with the scale parameter until the figures looked right. Posting this here in case it's useful to other people.
